src/planscape/plan/tests.py

```diff
@@ -1426,11 +1428,12 @@ def test_get_scenario_ok(self):
         self.assertEqual(scenario['notes'], 'my note')
         self.assertEqual(scenario['priorities'], {
                          'cond1': 2, 'cond2': 3})
-        self.assertEqual(scenario['project_areas'][str(
-            self.project_area.pk)]['geometry'], self.geometry)
-        self.assertEqual(scenario['project_areas'][str(
-            self.project_area.pk)]['properties']['estimated_area_treated'], 200)
         self.assertEqual(scenario['config']['max_budget'], 100)
+        scenario_output = scenario['ranked_project_areas'][str(
```

Comment:
What does the output look like when there are 2 project areas?
Can "rank" and "weighted_score" be present, too?

Reply:
done!